test/fuzz/test_runner.py

```diff
@@ -12,6 +12,10 @@
 import subprocess
 import logging
 
+# Fuzzers known to lack a seed corpus in https://github.com/bitcoin-core/qa-assets/tree/master/fuzz_seed_corpus
+FUZZERS_MISSING_CORPORA = [
+    "parse_iso8601",
+]
```

Comment:
Can remove this?

Also: (I know it is not related to this pull request), but could you please add `undefined` to `--with-sanitizers=fuzzer,address` in the `doc/fuzzing.md` readme?

Reply:
btw, the seed look rather boring this time: https://github.com/bitcoin-core/qa-assets/commit/0710bfd617ed8cbb337b7be79efba771632e9b1f

Reply:
Done! Please re-review :)